accelerator/config.h

```diff
@@ -28,6 +28,11 @@ extern "C" {
 
 #define TA_VERSION "tangle-accelerator/0.5.0"
 #define TA_HOST "localhost"
+#ifdef MQTT
+#define HOST "localhost"
+#define TOPIC "NB/test/room1"
```

Comment:
Why `room1` and `room2`? Can you use other mnemonic identifiers?

Reply:
I have changed them in the coming commit. They were testing topics. The new topics for subscribing and publishing follow the formats I presented yesterday.